rpc/src/rpc.rs

```diff
@@ -429,11 +465,18 @@ impl JsonRpcRequestProcessor {
         })?;
         let encoding = encoding.unwrap_or(UiAccountEncoding::Binary);
 
-        let response = get_encoded_account(&bank, pubkey, encoding, data_slice)?;
```

Comment:
looks like we can shrink the diff quite a bit by passing the runtime into `get_encoding_account()` and doing all the async magick down there?